src/Service.Tests/SqlTests/GraphQLPaginationTests/GraphQLPaginationTestBase.cs

```diff
@@ -212,14 +213,16 @@ public async Task RequestItemsOnly()
             }
 
             string graphQLQuery = @"{
-                supportedTypes(first: 3," + $"after: \"{after}\" " +
+                supportedTypes(first: 2," + $"after: \"{after}\" " +
```

Comment:
why does `first` need to be changed from 3 to 2?

Reply:
so that we can have tests that both land on the last page and land before the next page. Since there are only 4 records to look at, we need a value <3 to have another page available and get a non null endCursor. This was easier than adding new records and doesnt impact the test in any other way.